Emby.Naming/Subtitles/SubtitleParser.cs

```diff
@@ -37,7 +37,9 @@ public SubtitleInfo ParseFile(string path)
                 IsForced = _options.SubtitleForcedFlags.Any(i => flags.Contains(i, StringComparer.OrdinalIgnoreCase))
             };
 
-            var parts = flags.Where(i => !_options.SubtitleDefaultFlags.Contains(i, StringComparer.OrdinalIgnoreCase) && !_options.SubtitleForcedFlags.Contains(i, StringComparer.OrdinalIgnoreCase))
+            var parts = flags.Where(i =>
+                    !_options.SubtitleDefaultFlags.Contains(i, StringComparer.OrdinalIgnoreCase) &&
+                    !_options.SubtitleForcedFlags.Contains(i, StringComparer.OrdinalIgnoreCase))
```

Comment:
```suggestion
            var parts = flags.Where(i => !_options.SubtitleDefaultFlags.Contains(i, StringComparer.OrdinalIgnoreCase)
                && !_options.SubtitleForcedFlags.Contains(i, StringComparer.OrdinalIgnoreCase))
```

Reply:
This seems nicer to me, do we not have any rules about it?

Reply:
I didn't get any warnings / build errors so I don't think there are any rules for it, will commit the change

Reply:
Do we have a line length limit? The text was auto wrapped for me because it's longer then the default Rider line length

Reply:
There is no line length limit set as far as I know. FYI We are using StyleCop, and if you want to see how the rules are configured, the ruleset is here: https://github.com/jellyfin/jellyfin/blob/master/jellyfin.ruleset

Reply:
I was mainly referring to the boolean operator location, I think it looks much better at the beginning of a line.

Reply:
> I was mainly referring to the boolean operator location, I think it looks much better at the beginning of a line.

Personally I like them at the end :P But yes, I think at the beginning of the line matches the style in the rest of the project

Reply:
Well I also hate the excessive amount of newlines we use :laughing: have to make sacrifices for a consistent style.